iscript/src/iscript/mac.py

```diff
@@ -1173,16 +1181,23 @@ def get_notarization_status_from_log(log_path):
     if langpack_apps:
         await sign_langpacks(config, key_config, langpack_apps)
         all_paths = filter_apps(all_paths, fmt="autograph_langpack", inverted=True)
+
+    # app
     await extract_all_apps(config, all_paths)
     await unlock_keychain(
         key_config["signing_keychain"], key_config["keychain_password"]
     )
     await update_keychain_search_path(config, key_config["signing_keychain"])
     await sign_all_apps(config, key_config, entitlements_path, all_paths)
 
+    # pkg
```

Comment:
Depending on how long things take, we may want to re-`unlock_keychain`, especially in the l10n case where we might be signing 5 different apps and 5 different pkgs? We may hit the 10m timeout.

Reply:
Looks like about 6 minutes between unlock for signing and end of pkg signing on try, so that's a good safeguard. I'll add it.